pkg/Microsoft.ML.AutoMLFeaturizers/Microsoft.ML.Mkl.Redist.nupkgproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <Authors>Intel</Authors>
```

Comment:
You need to rename this file to be the name of the NuGet package.

Also, remove this Authors property.

Reply:
I fixed the files. It is creating separate nuget packages now in the bin/packages folder. It is not creating a folder under packages in the root though. Should it be creating them there as well?

Reply:
It's a little confusing, but `$RepoRoot\packages` are the NuGet packages that our code depends on - what are downloaded when we do a restore.  `$RepoRoot\bin\packages` are the NuGet packages that our build produces.

Reply:
Ok, that makes sense. I have the `$RepoRoot\bin\packages` creating the NuGet package correctly and tested it with a local import into another solution. So we should be good then. Thanks for the instructions!